.changeset/slimy-llamas-search.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lvfs/plugin-sonarqube': patch
```

Comment:
Think that this should be `'@backstage/plugin-sonarqube': patch`

Reply:
Whoops, will change this, thanks!